builder/Makefile

```diff
@@ -15,14 +15,18 @@ prelastword = $(word $(words $1),_ $1)
 cname_version = $(call lastword,$(subst -, ,$1))
 cname_arch = $(call prelastword,$(subst -, ,$1))
 
-define require_var =
-ifndef $1
-$$(error '$1 undefined')
+ifeq ($(REPO),)
+        override REPO := http://deb.debian.org/debian
+endif
+ifeq ($(COMMIT),)
+        override COMMIT := $(shell [ -z "$$(git status --porcelain 2> /dev/null)" ] && git rev-parse HEAD 2> /dev/null || echo local)
```

Comment:
this won't work as the makefile is run inside the builder container with only the features sub-directory of the config directory mounted, so we can not simply access the git status of the config repo here.